src/newUi.scss

```diff
@@ -2343,7 +2351,7 @@ nav {
   flex-direction: row;
   gap: 2px;
   align-items: center;
-  font-size: 14px;
+  font-size: 16px;
```

Comment:
The original design has both this and the timestamp column as 14px. However, we currently use 14px for this and 16px for the timestamp. Since both can use 16px, it's better for usability.